src/components/flowchart/flowchart.js

```diff
@@ -448,6 +449,44 @@ export class FlowChart extends Component {
     node && this.showTooltip(event, node.fullName);
   };
 
+  /**
+   * Enable a layer's active state when hovering it, update labelName's active className accordingly
+   * @param {Object} event Event object
+   * @param {Object} node Datum for a single node
+   */
+  handleLayerMouseOver = (event, node) => {
+    node &&
+      this.setState({
+        activeLayer: node.name,
+      });
```

Comment:
This is just a conditional, right? It's a bit off from our usual pattern, which would be this: 
```suggestion
    if (node) {
      this.setState({
        activeLayer: node.name,
      });
    }
```

Let me know if there's a good reason to do it this way though.

Reply:
hey yeah it's just a conditional. There's no good reason to do it this way 😄 I'll update the code to make it consistent with the codebase